### PR TITLE
Add Deta / PaaS

### PR DESCRIPTION
**Pull request by BetaHuhn**
Deta: "The Cloud for Doers & Dreamers."

Website: https://deta.sh
GitHub: https://github.com/deta
Twitter: https://twitter.com/detahq

[Deta Slack](https://deta-hq.slack.com/) has 19k members.

A few websites using Deta:

- https://webcrate.app
- https://github.com/sampoder/berowra
- https://github.com/berrysauce/microletter
- https://github.com/xeust/Method-Draw
- https://github.com/viperadnan-git/tiiny

More examples: [Deta GitHub Topic](https://github.com/topics/deta) or [awesome-deta](https://github.com/deta/awesome-deta)

### CODE REVIEW

File: src/technologies/d.json [hunk 1]
```diff
@@ -30,6 +30,17 @@
     },
     "website": "https://derak.cloud"
   },
+  "Deta": {
+    "cats": [
+      62
+    ],
+    "headers": {
```
Comment (nurbek91) [inline src/technologies/d.json:37]: "description": "Deta is a cloud platform for building and deploying app."

File: src/technologies/d.json [hunk 2]
```diff
@@ -30,6 +30,17 @@
...
+      "Server": "^Deta$"
+    },
+    "url": "^https?://[^/]+\\.deta\\.(?:app|dev)",
+    "icon": "deta.svg",
```
Comment (nurbek91) [inline src/technologies/d.json:41]: [Deta.svg.zip](https://github.com/AliasIO/wappalyzer/files/8352314/Deta.svg.zip)
32x32

File: src/technologies/d.json [hunk 3]
```diff
@@ -30,6 +30,17 @@
...
     "website": "https://derak.cloud"
   },
+  "Deta": {
+    "cats": [
+      62
+    ],
+    "headers": {
+      "Server": "^Deta$"
+    },
+    "url": "^https?://[^/]+\\.deta\\.(?:app|dev)",
```
Comment (nurbek91) [inline src/technologies/d.json:40]: example https://t1vm5r.deta.dev/

**Review by nurbek91 — APPROVED** [review]
Hey @BetaHuhn , thanks for the PR!!
Requires minor modifications to make it perfect.
I also wanted to ask if you have information about pricing.

File: src/technologies/d.json (hunk 3 above)
Comment (BetaHuhn) [reply src/technologies/d.json:40]: Not sure I understand what I need to change. Is the regex wrong?

**Comment by BetaHuhn** [comment]
> I also wanted to ask if you have information about pricing.

Deta is currently free for everyone, see [docs](https://docs.deta.sh/docs/home/#why-is-deta-cloud-free-for-ever).

File: src/technologies/d.json (hunk 3 above)
Comment (nurbek91) [reply src/technologies/d.json:41]: @BetaHuhn , nvr, I was looking for examples for this regex.

Comment (BetaHuhn) [reply src/technologies/d.json:41]: Ah okay :)

**Review by nurbek91 — APPROVED** [review]
Thanks!

**Pull request merged**